scipy/sparse/linalg/_special_sparse_arrays.py

```diff
@@ -0,0 +1,380 @@
+import numpy as np
+from scipy.sparse.linalg import LinearOperator
+from scipy.sparse import kron, eye, dia_array
+
+__all__ = ['LaplacianNd']
+
+
+class LaplacianNd(LinearOperator):
+    """
+    The grid Laplacian in `N` dimensions and its eigenvalues.
```

Comment:
Nit: we are inconsistent with the number of backticks used for N. Here and line 12, 19 one is used but line 56 uses 2. Since this isn't a variable I guess it should probably be 2.